src/lib/wind_estimator/WindEstimator.cpp

```diff
@@ -211,15 +211,15 @@ WindEstimator::run_sanity_checks()
 		}
 	}
 
-	if (!PX4_ISFINITE(_state(INDEX_W_N)) || !PX4_ISFINITE(_state(INDEX_W_E)) || !PX4_ISFINITE(_state(INDEX_TAS_SCALE))) {
+	if (!_state.isAllFinite()) {
 		_initialised = false;
 		return;
 	}
 
 	// attain symmetry
 	for (unsigned row = 0; row < 3; row++) {
 		for (unsigned column = 0; column < row; column++) {
-			float tmp = (_P(row, column) + _P(column, row)) / 2;
+			const float tmp = (_P(row, column) + _P(column, row)) * 0.5f;
```

Comment:
We should get rid of the _ + uppercase variables here as well.

https://github.com/PX4/PX4-Autopilot/pull/21013